UI/PlayerListWnd.cpp

```diff
@@ -21,6 +21,16 @@
 namespace {
     const int           DATA_PANEL_BORDER = 1;
 
+    std::string ColorToRgbTag(const GG::Clr color) {
```

Comment:
Rather move to Empire::RgbaColorTag()?

Reply:
Colour tags are not empire-related, so no.

Reply:
Can you think of some other suitable place for this code so it doesn't need to be reproduced in any .cpp file it is needed?

Reply:
Just use `GG::RgbaTag`